src/GitElephant/Command/BaseCommand.php

```diff
@@ -163,7 +169,7 @@ protected function addCommandName($commandName): void
      *
      * @return string
      */
-    protected function getCommandName(): string
+    protected function getCommandName(): ?string
```

Comment:
Actually, we can restrict methods calls on uninitialized properties... So even if a method could return "null", sometimes  we should restrict calling it, for example by prohibiting null as a returned type.
Not sure if this applies here, but just as a thought.